environments/site/tofu/node_group/variables.tf

```diff
@@ -201,3 +201,20 @@ variable "config_drive" {
     type = bool
     default = false
 }
+
+variable "additional_cloud_config" {
```

Comment:
Can we also set `nullable = false` on this please? Then if it is not defined in the module inputs, we get the default. Strictly, it is a bit unnecessary given we do a lookup for module inputs defaulting to the top-level var (which has a default) but it is a guard against messing that up at some point.